modules/services/eks/iam.tf

```diff
@@ -0,0 +1,72 @@
+// This is a Single Account installation. The resources are created globally (instead of regionally). 
+data "aws_iam_policy_document" "ecr_pull_image" {
+  count = local.n
```

Comment:
Do we actually need this data to be conditional with count? We can simplify the usage by not specifying a count, since it doesn't create any resource.